policy/scan/local_scanner.go

```diff
@@ -301,9 +301,21 @@ func (s *LocalScanner) distributeJob(job *Job, ctx context.Context, upstream *up
 		return nil, false, nil
 	}
 
+	runtimeEnv := execruntime.Detect()
+	var runtimeLabels map[string]string
+	if runtimeEnv != nil {
+		runtimeLabels = runtimeEnv.Labels()
+	}
+
 	justAssets := []*inventory.Asset{}
 	for _, asset := range assets {
 		asset.asset.KindString = asset.asset.GetPlatform().Kind
+		for k, v := range runtimeLabels {
```

Comment:
do we not wanna check if its an automated env before applying all these? (saw that we have that check in v8)

Reply:
`	if opts.AutoDetectCICDCategory && runtimeEnv.IsAutomatedEnv() || opts.Category == "cicd" {`

Reply:
we cannot check this in the same way in this part of the code. I am now checking if the root asset is ci/cd. This should indicate that we are doing a ci/cd scan